rfcs/react-components/components/dev-docs.md

```diff
@@ -0,0 +1,66 @@
+As a new developer to the Fluent repo, but long time Fluent consumer and fan, the supporting documentation has been a rough experience. This has forced me to randomize other members of the team which slows us both down. This RFC seeks to lay out a plan to modernize the Fluent dev docs.
+
+## Current issues:
+
+- Mixture of v6, 7, 8 and 9 specific docs.
+- Docs seemed to have stopped being updated in the early days of v9 development.
+- Mixture of process, set up, and reference docs.
+- RFC’s seem to be a large part of the Fluent process now, but there’s no clear way for an outsider to observe or contribute to them.
+- Use of Github’s wiki isn't the best technical solution.
+
+## Goals of the new system:
+
+- Be more accessible to developers in their work flow.
+- Help developers onboard faster without distracting core team members.
```

Comment:
Do we want to throw any target metrics or times here? In the MADS repo we had the 10 minute goal from pulling the repo to being product even for new developers. Not saying we should use that goal but having that kind of a target for what we're doing might be a good idea.

Reply:
Good callout. But I do want to be cautious about scope creep, so leaving out discussion of how to improve our dev infrastructure from this RFC.

Reply:
Yah it was more about the spirit of it...so something like "as a new Fluent developer I can answer questions around making a new component in 2 minutes with existing docs" or something (don't copy that just threw it out there :D)